javascript/chronometer.js

```diff
@@ -1,39 +1,64 @@
 // Constructor
-// function Chronometer() {
+function Chronometer() {
+    this.currentTime = 0;
+    this.intervalId = '';
+    this.minutes = 0;
+    this.seconds = 0;
+    this.currentMilliseconds = 0;
+    this.milliseconds = 0;
+}
 
-// }
+Chronometer.prototype.startClick = function () {
+    this.intervalId = setInterval(() => {
+        this.setTime()
+        this.currentTime++;
+        printTime();
+    }, 1000);
+    this.scdIntervalId = setInterval(() => {
+        this.setTime()
+        this.currentMilliseconds++;
```

Comment:
Instead of having two intervals you could use only one for the milliseconds and add a second every 100 loops of the millisecond function.